bf:luis convert error while generating .luis file

## Versions
@microsoft/botframework-cli/4.10.0 win32-x64 node-v12.18.1
PowerShell 7.0.3
Windows 10 x64
botskills version 1.0.16

## Describe the bug
While generating the .luis file using the **bf:luis convert** command, I get the following error:
`[ERROR] line 955:19 - line 955:22: syntax error: no viable alternative at input ' All'
[ERROR] line 955:19 - line 955:22: syntax error: invalid input 'All' detected. Expecting one of this - end of file, Whitespace, new line, undefined, model description, QnA definition, Intent definition, undefined, undefined, undefined, undefined`

The input .lu file,  the line in question has the following:
`@ intent Top Ideas All Ideations usesFeature topIdeasAccrossIdeations`

When I changed the name of the intent in the LUIS portal to exclude whitespaces, then the command ran successfully. This shows that if an intent is named in LUIS with whitespaces between words, then the **convert** command is failing

## To Reproduce
Steps to reproduce the behavior:
1. Create an intent with whitespaces in luis portal
2. Run the bf:luis convert command to generate the .luis file from .lu file
3. Notice error

## Expected behavior
The command must consider intents that may have been named with whitespaces and hence got updated in the .lu file as well with whitespaces while considering the generation of the .luis file from .lu file

## Screenshots
Error when running the command.
![image](https://user-images.githubusercontent.com/29098159/90533820-a2268980-e196-11ea-9358-75cb1548ec9b.png)

The section of the LU file in question
![image](https://user-images.githubusercontent.com/29098159/90533999-d8640900-e196-11ea-99a4-f099447382cd.png)

## Additional context
I originally encountered this error while running botskills update, which seems to internally call the bf:luis convert command. Tried it as a standalone and found the same.

Was able to fix by changing the intent name in the LUIS portal, but this is just a workaround.

Post interim fix:
![image](https://user-images.githubusercontent.com/29098159/90534302-3690ec00-e197-11ea-9d23-f9d81087e920.png)


[bug]

--- a/packages/lu/src/parser/luis/luConverter.js
+++ b/packages/lu/src/parser/luis/luConverter.js
@@ -72,7 +72,7 @@ const parseIntentsToLu = function(luisObj, luisJSON){
         if (intent.intent.features) {
             let rolesAndFeatures = addRolesAndFeatures(intent.intent);
             if (rolesAndFeatures !== '') {
-                fileContent += `@ intent ${intent.intent.name}`;
+                fileContent += `@ intent ${intent.intent.name.includes(' ') ? `"${intent.intent.name}"` : `${intent.intent.name}`}`;
                 fileContent += rolesAndFeatures;
                 fileContent += NEWLINE + NEWLINE;
             }

--- a/packages/lu/test/fixtures/testcases/intentWithSpace.json
+++ b/packages/lu/test/fixtures/testcases/intentWithSpace.json
@@ -1,0 +1,39 @@
+{
+  "intents": [
+    {
+      "name": "test intent",
+      "features": [
+        {
+          "modelName": "bar",
+          "isRequired": false
+        }
+      ]
+    }
+  ],
+  "entities": [
+    {
+      "name": "bar",
+      "roles": []
+    }
+  ],
+  "composites": [],
+  "closedLists": [],
+  "regex_entities": [],
+  "regex_features": [],
+  "utterances": [
+    {
+      "text": "foo",
+      "intent": "test intent",
+      "entities": []
+    }
+  ],
+  "patterns": [],
+  "patternAnyEntities": [],
+  "prebuiltEntities": [],
+  "luis_schema_version": "7.0.0",
+  "versionId": "0.1",
+  "name": "",
+  "desc": "",
+  "culture": "en-us",
+  "phraselists": []
+}

--- a/packages/lu/test/parser/luis/luisBuilder.test.js
+++ b/packages/lu/test/parser/luis/luisBuilder.test.js
@@ -72,5 +72,11 @@ assert.isTrue(luisObject.validate())
         assert.equal(luisObject.content.includes(`- {@add=add {@globalCount={@count={@countNumber=two} apples}}}`), true);
     })
 
+    it('Intent name with spaces are handled correctly with feature assignment', async () => {
+        let testJSON = require('../../fixtures/testcases/intentWithSpace.json');
+        const luisObject = LUISBuilder.fromJson(testJSON).parseToLU();
+        assert.equal(luisObject.content.includes(`@ intent "test intent" usesFeature bar`), true);
+    })
+
     
 });

